src/pages/BookingServicePage/BookingServicePage.jsx

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import BookingServiceFormContainer from "../../containers/BookingServiceForm/BookingServiceForm.container";
+import MobileNavigationBar from "../../containers/MobileNavigationBar/MobileNavigationBar.container";
 
 const BookingServicePage = () => {
   return (
-    <div className="page" style={{ marginTop: "2rem" }}>
-      <BookingServiceFormContainer />
-    </div>
+    <>
+      <MobileNavigationBar title="Teethcare" />
```

Comment:
actualy...this should be in AppRouter? you only need to check if the role is patient and then display it lmao, no need to put in each page like this